### ISSUE TITLE
FileEdit widget error with mode='d'

### ISSUE THREAD

**Issue by uschmidt83** [napari/magicgui#156]
**Describe the bug**
An error occurs when using `mode='d'` with the `FileEdit` widget.

**To Reproduce**
```python
from magicgui import magicgui

@magicgui(dir_picker = dict(widget_type='FileEdit', mode='d'))
def func(dir_picker):
    pass

func.show()
```
Pressing the `Choose directory` button.

**Screenshots**
![image](https://user-images.githubusercontent.com/231996/108701360-207a3480-7508-11eb-9186-1b4c4b73019f.png)

**Environment:**
 - OS: Ubuntu 20.04
 - `qtpy.PYQT_VERSION` 5.15.2
 - `magicgui.__version__` 0.2.6

**Comment by tlambert03** [comment]
Thanks @uschmidt83! fixed in #158

**Comment by tlambert03** [comment]
I'll push a release with this fix and some other things you've been seeing within the next day or two
